docs/api/rest_api/rest_api_usage/testing_rest_api.md

```diff
@@ -23,23 +23,19 @@ For examples of using `curl`, refer to:
 
 ## PHP
 
-To test REST API with PHP, cURL can be used. Open a PHP shell in a terminal with `php -a` and copy-paste this code into it:
-
-```php
-$resource = 'https://api.example.com/api/ibexa/v2/content/objects/52';
-$curl = curl_init($resource);
-curl_setopt_array($curl, [
-    CURLOPT_HTTPHEADER => ['Accept: application/vnd.ibexa.api.ContentInfo+json'],
-    CURLOPT_HEADERFUNCTION => function($curl, $header) {
-        if (!empty($cleanedHeader = trim($header))) {
-            var_dump($cleanedHeader);
-        }
-        return strlen($header);
-    },
-    CURLOPT_RETURNTRANSFER => true,
-]);
-var_dump(json_decode(curl_exec($curl), true));
-```
+To test REST API with PHP, [cURL](https://www.php.net/manual/en/book.curl.php) or the [Symfony HttpClient](https://symfony.com/doc/5.4/http_client.html) can be used. Open a PHP shell in a terminal with <nobr>`php -a`</nobr> and copy-paste this code into it:
```

Comment:
```suggestion
You can use [cURL](https://www.php.net/manual/en/book.curl.php) or the [Symfony HttpClient](https://symfony.com/doc/5.4/http_client.html) to test REST API with PHP .
Open a PHP shell in a terminal with <nobr>`php -a`</nobr> and copy-paste this code into it:
```

Reply:
Thank you @MagdalenaZuba 
I was searching a way to emphase that cURL is built in PHP while HttpClient is part of third party Symfony. I couldn't do it with previous sentence construction.

```suggestion
You can use [PHP cURL](https://www.php.net/manual/en/book.curl.php) or [Symfony HttpClient](https://symfony.com/doc/5.4/http_client.html) to test REST API.
Open a PHP shell in a terminal with <nobr>`php -a`</nobr> and copy-paste this code into it:
```